src/screens/Assets/liquidityPools/LiquidityPoolsTab.js

```diff
@@ -92,13 +92,21 @@ function LiquidityPoolsTab() {
     return <ChainListHeader chain={chain} balance={balance} onPress={() => toggleExpandItems(chain)} />;
   };
 
-  const renderItem = (headerListItem: HeaderListItem<LiquidityPoolItem>) => {
+  const renderItem = (headerListItem: HeaderListItem<LiquidityPoolAssetBalance>) => {
     if (headerListItem.type === 'header') {
       return <ServiceListHeader title={headerListItem.key} />;
     }
 
-    const { title, iconUrl, value, change } = headerListItem.item;
-    return <LiquidityPoolListItem title={title} iconUrl={iconUrl} value={value} change={change} />;
+    const { title, iconUrl, value, change, share } = headerListItem.item;
+    return (
+      <LiquidityPoolListItem
+        title={title}
+        subtitle={share ? t('poolShare', { share }) : null}
```

Comment:
Pool share could be formatted in LiquidityPoolListItem to match behavior of passing other values as raw values rather than formatted ones.

Reply:
My idea behind this is to separate it in case we want to add different subtitle for different `LiquidityPool`